src/events/interactionCreate.ts

```diff
@@ -87,6 +87,25 @@ export const execute = async (interaction: Interaction) => {
           .setDescription(`The tag \`${name}\` has been added successfully`);
         await interaction.reply({ embeds: [embed], ephemeral: true });
       }
+      if (interaction.customId === 'tagEditForm') {
```

Comment:
Add a check here for if the user still has permissions

Reply:
:+1: